frontend/src/static/js/api.js

```diff
@@ -33,6 +33,7 @@ window.api = {
     return loadJSON(`${REPORT_DIR}/summary.json`)
         .then((data) => {
           window.app.creationDate = data.dashboardGeneratedTime;
+          window.app.sinceDate = data.repos[0].sinceDate;
```

Comment:
perhaps with this addition, let have the since date of all repos be a direct element in `summary.json`.

Reply:
Okay, I think better to have that in a separate PR

Reply:
>Okay, I think better to have that in a separate PR

Alright, please open an issue on that and reference to this PR